subprojects/docs/src/main/resources/header.html

```diff
@@ -165,50 +164,98 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
-                    <li><a href="ant.html">Using Ant from Gradle</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#declaring-dependency-versions" aria-expanded="false" aria-controls="declaring-dependency-versions">Choosing Versions</a>
+                <ul id="declaring-dependency-versions">
+                    <li><a href="single_versions.html">Declaring a Simple Version</a></li>
+                    <li><a href="rich_versions.html">Declaring a Rich Version</a></li>
+                    <li><a href="dynamic_versions.html">Handling Changing Versions</a></li>
+                    <li><a href="dependency_locking.html">Locking Versions</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Example Gradle Projects</a>
-                <ul id="sample-gradle-builds">
-                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL Samples</a></li>
-                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL Samples</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#controlling-transitive-dependencies" aria-expanded="false" aria-controls="controlling-transitive-dependencies">Controlling Transitives</a>
+                <ul id="controlling-transitive-dependencies">
+                    <li><a href="dependency_constraints.html">Declaring Constraints</a></li>
+                    <li><a href="platforms.html">Using Platforms</a></li>
+                    <li><a href="dependency_alignment.html">Aligning Dependencies</a></li>
+                    <li><a href="dependency_capability_conflict.html">Managing Conflicting Dependencies</a></li>
+                    <li><a href="component_metadata_rules.html">Fixing Metadata</a></li>
+                    <li><a href="resolution_rules.html">Customizing Resolution</a></li>
+                </ul>
+            </li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#modeling-features" aria-expanded="false" aria-controls="modeling-features">Utilizing Variants</a>
+                <ul id="modeling-features">
+                    <li><a href="component_capabilities.html">Declaring Capabilities</a></li>
+                    <li><a href="feature_variants.html">Modeling Feature Variants</a></li>
+                    <li><a href="variant_model.html">Understanding Variant Selection</a></li>
+                    <li><a href="cross_project_publications.html">Adding Variants to Subprojects</a></li>
+                </ul>
+            </li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#publishing" aria-expanded="false" aria-controls="publishing">Publishing Modules</a>
+                <ul id="publishing">
```

Comment:
I had moved back _publishing_ one level up. It really does not fit inside _dependency management_, it is an orthogonal concept.

So I think this location is wrong as well. It could be placed as a higher level node between JVM and native sections.

Reply:
As discussed, we keep it where it is, but renamed the whole chapter to make everything fit in:

*Managing Dependencies* -> *Working with Dependencies*